designdocs/privacy-design.md

```diff
@@ -0,0 +1,163 @@
+
+#### User Communication
+The judgement on how to communicate to the user the known threats is up to the implementer. It is suggested that the following threat vectors be communicated to the user at the time of session creation:
+
+<table>
+  <tr>
+   <td><strong>Scenario</strong>
+   </td>
+   <td><strong>Threat Vector</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>XRSession can create an <a href="https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-unbounded">unbounded</a> reference space.
+   </td>
+   <td>Site may be able to determine user’s specific geographic <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-location">location</a>, and may be able to perform gait analysis, allowing user <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-profiling">profiling</a> and <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-fingerprinting">fingerprinting</a>.
+   </td>
+  </tr>
+  <tr>
+   <td>XRSession can create a <a href="https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-floor-level">floor-level</a> or <a href="https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-bounded">bounded</a> reference space, and the origin of the reference space will reflect the real-world location of the user’s floor.
+   </td>
+   <td>Site may be able to infer the user’s height and may be able to perform gait analysis, allowing user <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-profiling">profiling</a> and <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-fingerprinting">fingerprinting</a>.
+   </td>
+  </tr>
+  <tr>
+   <td>XRSession may provide <a href="https://immersive-web.github.io/webxr/#xrviewerpose">XRViewerPose</a> data that is not quantized, and underlying device sensors may be susceptible to fingerprinting.
+   </td>
+   <td>Site may be able to perform user <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-fingerprinting">fingerprinting</a>.
+   </td>
+  </tr>
+  <tr>
+   <td>On a device with configurable interpupillary distance, XRSession may create an <a href="https://immersive-web.github.io/webxr/#xrviewerpose-interface">XRViewerPose</a> that will return an <a href="https://immersive-web.github.io/webxr/#xrview">XRView</a> for each eye
+   </td>
+   <td>Site access to IPD data may allow user <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-profiling">profiling</a> and <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-fingerprinting">fingerprinting</a>.
+   </td>
+  </tr>
+</table>
+
+#### Camera Data and XRSession Data
```

Comment:
I'm not really sure this section is accomplishing anything since the underlying tracking systems use that same camera data along with IMU data to perform the combined pose estimation.  There's no reason a webxr polyfill wouldn't do the same thing (and in fact do), so I'm not sure what this restriction gains in terms of privacy?